boards/feather_m0/Cargo.toml

```diff
@@ -47,6 +47,11 @@ optional = true
 version = "0.1"
 optional = true
 
+[dependencies.rtic-monotonic]
+git = "https://github.com/rtic-rs/rtic-monotonic"
```

Comment:
git dependencies can't be published on crates.io

Reply:
It's probably a good idea to wait for `rtic-monotonic` and RTIC 0.6 to be out of alpha before merging this PR anyway.

Reply:
what about 

```
[dependencies.dwt-systick-monotonic]
version = "0.1.0-alpha.1"
```

For reference: https://github.com/rtic-rs/cortex-m-rtic/blob/master/examples/periodic.rs#L13-L17

Reply:
Changed to pull from "v0.1.0-alpha.0"

https://crates.io/crates/rtic-monotonic